src/utils/pr.test.ts

```diff
@@ -0,0 +1,36 @@
+import { describe, it, expect } from "vitest";
+import { getCheckboxValueFromString } from "./pr";
+
+describe("pr", () => {
+  it("should check for the correct checkbox value", () => {
+    const prDescription = `
+This PR was opened by the [ready-release-go](https://github.com/woodpecker-ci/plugin-ready-release-go) plugin.
```

Comment:
I always wonder if "ready-release-go" is the best possible name for the plugin 😅

Reply:
Dunno, isn't it fitting. You wait til it's ready, a release will be created and you are ready to go with using the new version 😉

Reply:
Well kind of, if you explain it like this sure. So far I just didn't read it like "ready -> release -> go" but couldn't really wrap my head around it 😅 I guess I would have simply named it `plugin-release-helper`. But then you have a broader scope than just WP so...